app/models/document_type.rb

```diff
@@ -35,6 +36,14 @@ def guidance_for(id)
     @guidance.find { |guidance| guidance.id == id }
   end
 
+  def hint_govspeak
+    begin
+      govspeak_to_html(I18n.t!("document_types.#{id}.guidance.body_govspeak"))
```

Comment:
Could we just use `t` (vs. `t!`)? I think that would remove the need for this change, although we'll probably need a variable in the view.

Reply:
Unfortunately not, as `t` silently fails and returns "translation missing: document_types.news_story.guidance.body_govspeak". The safest way to force a `nil` return is to throw an exception via `t!` and catch it.

Reply:
Note to self after call with Ben: this should be accomplishable with something like `t.has_key?("...")` rather than relying on exceptions. (Will certainly make the code nicer!)